spec/models/user_spec.rb

```diff
@@ -3,10 +3,6 @@
 RSpec.describe User do
   describe "attribute validations" do
     it { is_expected.to validate_presence_of(:username) }
-    it { is_expected.to validate_presence_of(:password) }
-    it { is_expected.to validate_length_of(:password) }
-    it { is_expected.to validate_presence_of(:password_confirmation) }
-    it { is_expected.to validate_length_of(:password_confirmation) }
```

Comment:
@jfranciscosousa do you recommend other kind of check or can I just remove it?

Reply:
Comment everything and leave a #TODO comment also, I'll take a look later